.github/workflows/pipeline.yaml

```diff
@@ -134,20 +140,37 @@ jobs:
           name: packaged-testing.yaml
           path: packaged-testing.yaml
 
+  build-and-package-for-prod:
```

Comment:
Prod & Dev environments should be in separate pipelines. 

Again, the general idea is that we should never have pipeline files that are too long. 

Especially when you named your file pipeline.yaml -> How am I, or your teammates, supposed to know what the pipeline is supposed to do?